vspace/src/main/webapp/resources/extra/Home.css

```diff
@@ -7,8 +7,10 @@
 	--base-grey: #F2F2F2;
 }
 
-i {
-	font-size: 20px;
+[class^="icon-"], [class*=" icon-"] {
+	font-size: 24px;
+	vertical-align: sub;	
+	cursor: pointer;

```

Comment:
that turns the cursor for all icons into a pointer, right? There might be icons though that are used just for layout purposes not to be clickable for which the pointer shouldn't change, so I wouldn't do that her.